sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/Rules/CorrelationRuleFilter.cs

```diff
@@ -25,7 +25,7 @@ namespace Azure.Messaging.ServiceBus.Administration
     /// </para>
     /// <para>
     /// The CorrelationRuleFilter provides an efficient shortcut for declarations of filters that deal only with correlation equality.
-    /// In this case the cost of the lexigraphical analysis of the expression can be avoided.
+    /// In this case the cost of the lexicographical analysis of the expression can be avoided.
```

Comment:
I'll admit.... I had to look this up and compare the terms since both are valid spellings.   Nice catch!

Reply:
I also had to look it up. I've heard 'lexigraphical' say verbally, but seen 'lexicographical' written before. Most dictionaries don't have 'lexigraphical'.